### PR TITLE
P0 avatar badge

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
Here I have added two components 
- Avatar
- badge
- also added relevant vars and class needed for the implementation

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Good job, Shivam! 

Looks good to merge!

**Review by ScoobyDubeyDo — APPROVED** [review]
👍

File: atoms/badge/index.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    </div>
+
+    <div class="tui__pos--rel">
+        <div
+        class="tui__badge-xs tui__bgcol--green tui__badge-pos-abs-tr">
+        44
+        </div>
+    </div>
+
+    <div class="tuid__badge--display tui__pos--rel">
```
Comment (SJTGSHIVAM) [inline atoms/badge/index.html]: I just depicted while reviewing a minor fix is needed here. As tuid__badge--display is not a class the users will be able to access so we should remove it from here.
Moreover, this class is not related to any functionality it is just there to give background color and padding to the representation div which is used as a placeholder for the application of badges.

Comment (SJTGSHIVAM) [reply atoms/badge/index.html]: I just made the changes that were needed

**Review by anuj-kumary — APPROVED** [review]

**Review by Hnh013 — APPROVED** [review]
avatars and badges look good 
a suggestion: if feasible try to use class for avatar's properties and for badge margin properties

**Comment by SJTGSHIVAM** [comment]
> avatars and badges look good a suggestion: if feasible try to use class for avatar's properties and for badge margin properties

 I believe I am missing something can you point me to the code where is the issue

**Review by varnita619 — APPROVED** [review]
Looks good to go

File: atoms/avatar/index.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  height: var(--avatar-height-xxl);
+  width: var(--avatar-width-xxl);
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+.tui__avatar-xl {
+  height: var(--avatar-height-xl);
+  width: var(--avatar-width-xl);
+  display: inline-flex;
```
Comment (Hnh013) [inline atoms/avatar/index.css]: if feasible , try to use a base/utility class

File: atoms/badge/index.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.tui__badge-xs {
+  padding: var(--padding-xs);
+  border-radius: var(--badge-border-radius-lg);
+}
+.tui__badge-pos-abs-tr {
+  position: absolute;
+  top: 0%;
```
Comment (Hnh013) [inline atoms/badge/index.css:7]: add position base/utility class if feasible

**Comment by SJTGSHIVAM** [comment]
@Hnh013 Thank you for pointing that out I have made the necessary changes

File: atoms/badge/index.css (hunk 3 above)
Comment (SJTGSHIVAM) [reply atoms/badge/index.css:7]: As discussed over the call these are absolute properties and we don't need to create a separate class as it as these values are specific to the utility class. In case we need to change these values here then we may have to reconsider having more utility classes to cater to the purpose itself.
